tests/tests_pytorch/strategies/test_ddp_hydra_support.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+import pytest
+
+from pytorch_lightning.utilities.imports import _RequirementAvailable
+from tests_pytorch.helpers.runif import RunIf
+
+_HYDRA_AVAILABLE = bool(_RequirementAvailable("hydra-core"))
```

Comment:
Why not use the variable from `src/pytorch_lightning/strategies/launchers/subprocess_script.py`?

Reply:
Are you suggesting I use `from pytorch_lightning.strategies.launchers.subprocess_script import _HYDRA_AVAILABLE`?

Reply:
Exactly!